common/kvstore/tablestore/table_view.go

```diff
@@ -0,0 +1,183 @@
+package tablestore
+
+import (
+	"encoding/binary"
+	"github.com/Layr-Labs/eigenda/common/kvstore"
+	"github.com/syndtr/goleveldb/leveldb/iterator"
+	"github.com/syndtr/goleveldb/leveldb/util"
+)
+
+var _ kvstore.Table = &tableView{}
+
+// tableView allows table in a New to be accessed as if it were a Store.
```

Comment:
The comment not readable/has typo ("in a New"?, "as it were a Store"?)

Reply:
```
// tableView allows data in a table to be accessed as if it were a Store.
```

This is an artifact of my IDE attempting to automatically refactor things when I changed the name of the method.